data/deep missions.txt

```diff
@@ -718,36 +720,35 @@ mission "Deep: Remnant Technology"
 		has "event: deep: scan log research"
 	on offer
 		payment 250000
-		dialog `You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and informed the Deep government of your recent help in this immense discovery. Please meet us on <destination> if you wish to assist us further. The help would be appreciated, but it is no longer required for us to continue our work."`
+		dialog `You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and made sure you were listed as a primary contributor on all the reports of these immense discoveries. Please meet us on <destination> if you wish to assist us further. Your help would be appreciated, but we will make progress even if you're a bit busy at the moment."`
 	
-	on decline
-		set "Deep: Remnant Research: done" # Allows the player to decline this and any mission beyond this point and still get a chance to unlock the Bactrian, but not doing the other half of the Deep: Remnant chain could ruin future situations for the player.
+	on fail
```

Comment:
`on fail` doesn't trigger if the player never accepted to mission to begin with, does it? 

Although, `set "Deep: Remnant Research: done"` should be triggered in both an `on fail` and `on decline` for this mission (assuming the player aborts it after accepting).